docs/environment.yml

```diff
@@ -4,6 +4,7 @@ name: jhub_docs
 channels:
   - conda-forge
 dependencies:
+- pip
```

Comment:
It's weird that it is not picking up pip on line 18 for installation.

Reply:
Well on line 18 it's a pip subsection, and that's because there is line 18 that it complains. 

The reason to have to include pip on it's own is that you may want to pin pip version I believe: 

```
- pip==18.3:
    - entrypoints
    ...
```

Would be weird, so you need:
```
  - pip 
  - pip:
      - entrypoints
```